### ISSUE TITLE
The type definition file is missing in published package.

### ISSUE THREAD

**Issue by a4180p** [shuhei/rolling-window#20]
Steps to reproduce:
```
mkdir /tmp/tst &&  \
cd /tmp/tst && \
yarn init && \
yarn add @shuhei/rolling-window && \
ls -la node_modules/@shuhei/rolling-window/dist
```

**Comment by shuhei** [comment]
Thanks for letting me know! Will take a look.

**Comment by shuhei** [comment]
For some reason, `yarn pack` ignores all the files in `dist` except for `dist/index.js` while `npm pack` doesn't. Will publish a new version with `npm publish` after a meeting today.

**Comment by shuhei** [comment]
@ponimas Published 0.2.1. It has the missing `dist` files.

```sh
[~/work/js/rolling-window] (master) € npm view @shuhei/rolling-window

@shuhei/rolling-window@0.2.1 | MIT | deps: none | versions: 11
A rolling time window

dist
.tarball: https://registry.npmjs.org/@shuhei/rolling-window/-/rolling-window-0.2.1.tgz
.shasum: 05f39bba8c176b0d24ec58530bea6f469ebc1960
.integrity: sha512-ZWWlP17yrhGob7BTwYnW60DSOZhYw1I/qsFcIylh8igGApBuXA5R47jKYacHnlDCSlhsuGLZZQdKDuhfDnXILw==
.unpackedSize: 26.2 kB

maintainers:
- shuhei <shuhei.kagawa@gmail.com>

dist-tags:
latest: 0.2.1

published 38 seconds ago by shuhei <shuhei.kagawa@gmail.com>
[~/work/js/rolling-window] (master) € wget https://registry.npmjs.org/@shuhei/rolling-window/-/rolling-window-0.2.1.tgz
--2020-08-20 17:05:40--  https://registry.npmjs.org/@shuhei/rolling-window/-/rolling-window-0.2.1.tgz
Resolving registry.npmjs.org (registry.npmjs.org)... 104.16.21.35, 104.16.17.35, 104.16.18.35, ...
Connecting to registry.npmjs.org (registry.npmjs.org)|104.16.21.35|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 7719 (7.5K) [application/octet-stream]
Saving to: ‘rolling-window-0.2.1.tgz’

rolling-window-0.2.1.t 100%[==========================>]   7.54K  --.-KB/s    in 0.002s

2020-08-20 17:05:41 (3.63 MB/s) - ‘rolling-window-0.2.1.tgz’ saved [7719/7719]

[~/work/js/rolling-window] (master*) € tar -tvf rolling-window-0.2.1.tgz
-rw-r--r--  0 0      0          20 Oct 26  1985 package/.prettierignore
-rw-r--r--  0 0      0        1070 Oct 26  1985 package/LICENSE
-rw-r--r--  0 0      0         522 Oct 26  1985 package/.eslintrc.js
-rw-r--r--  0 0      0         227 Oct 26  1985 package/dist/index.js
-rw-r--r--  0 0      0         269 Oct 26  1985 package/jest.config.js
-rw-r--r--  0 0      0        3414 Oct 26  1985 package/dist/RollingWindowHistogram.js
-rw-r--r--  0 0      0        1153 Oct 26  1985 package/package.json
-rw-r--r--  0 0      0        5818 Oct 26  1985 package/tsconfig.json
-rw-r--r--  0 0      0         309 Oct 26  1985 package/CHANGELOG.md
-rw-r--r--  0 0      0        2787 Oct 26  1985 package/README.md
-rw-r--r--  0 0      0          98 Oct 26  1985 package/dist/index.d.ts
-rw-r--r--  0 0      0         102 Oct 26  1985 package/src/index.ts
-rw-r--r--  0 0      0        2094 Oct 26  1985 package/dist/RollingWindowHistogram.d.ts
-rw-r--r--  0 0      0        3758 Oct 26  1985 package/src/__tests__/RollingWindowHistogram.ts
-rw-r--r--  0 0      0        3581 Oct 26  1985 package/src/RollingWindowHistogram.ts
-rw-r--r--  0 0      0         985 Oct 26  1985 package/.github/workflows/nodejs.yml
```

**Comment by shuhei** [comment]
I could remove some unnecessary files from the package though 😅
